packages/affiliates/payouts/dappmining/2021-02-01_2021-02-08_YD-BTC-MAR21_0004.json

```diff
@@ -5,10 +5,7 @@
     "empAddress": "0x1c3f1A342c8D9591D9759220d114C685FD1cF6b8",
     "totalRewards": 4595,
     "defaultAddress": "0xa0dfF1F09a0DCEda7641b7eB07c3BDbe66a06C35",
-    "whitelist": [
```

Comment:
I guess these changes were never linted before that's why they're included in diff.

Reply:
ye, I had the same diff one of my other PRs.